sourabh/To Do List.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
you didnt write here "html" tag, but you've wrote it in other files, why is that?